.env

```diff
@@ -0,0 +1,13 @@
+MYSQLDB_USER=root
```

Comment:
real .env should never in source control, add it to .gitignore, use git rm --cached .env, add .env.template with actual values removed

Reply:
i will remember it